### PR TITLE
Add codestart for spring-web

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by aureamunoz** [comment]
@ia3andy You can review now :-)

**Review by ia3andy — APPROVED** [review]
Awesome!

**Comment by ia3andy** [comment]
cc @aloubyansky @geoand

**Comment by geoand** [comment]
Great, thanks a lot!

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/spring-web-example/codestart.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+language:
+  base:
+    data:
+      resource:
+        class-name: ExampleController
+        path: "/springweb/hello"
+        response: "hello"
+      package-name: org.acme.spring.web
+    dependencies:
+      - io.quarkus:quarkus-resteasy
```
Comment (aloubyansky) [inline devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/spring-web-example/codestart.yml:14]: resteasy can be removed, quarkus-spring-web already depends on it.

**Comment by aloubyansky** [comment]
Thanks!

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/spring-web-example/codestart.yml (hunk 1 above)
Comment (aureamunoz) [reply devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/spring-web-example/codestart.yml:14]: :grimacing:

**Pull request merged**